wellcomeml/ml/bert_classifier.py

```diff
@@ -114,7 +116,8 @@ def fit(self, X, Y):
            Y: list or numpy array of classes (n_samples, n_classes)
         """
         num_labels = len(Y[0])
-        self._init_model(num_labels)
+        if not self.initiated_:
```

Comment:
good addition, you could also check for existence of model and tokenizer but this works as well